pkgs/development/python-modules/lektor/default.nix

```diff
@@ -37,6 +38,8 @@ buildPythonPackage rec {
     flask pyopenssl ndg-httpsclient
   ] ++ lib.optionals isPy27 [ functools32 ];
 
+  pythonPath = [ setuptools ];
```

Comment:
setuptools is available during build/installation. This needs to be exposed at runtime
```suggestion
```

Reply:
Changed with https://github.com/NixOS/nixpkgs/compare/e4abda30967e1d949ce403613af83f6ef0566feb..07ba44d1b689c17c486e758002231d1c5dbf9f50